.gitignore

```diff
@@ -23,3 +23,4 @@
 .suo
 *.VC.db
 *.VC.opendb
+srcDoc/doc
```

Comment:
I believe this should be doc. We want to generate the documentation in ./doc, not ./srcDoc/doc

Reply:
done